upgrade/upgrade-provider.go

```diff
@@ -187,6 +187,24 @@ func UpgradeProvider(ctx Context, repoOrg, repoName string) error {
 			}).AssignTo(&tfSDKUpgrade),
 		)
 	}
+	if ctx.UpgradePfVersion {
+		discoverSteps = append(discoverSteps,
+			step.F("Planning Plugin Framework Update", func() (string, error) {
+				latest, err := latestRelease(ctx, "pulumi/pulumi-terraform-bridge")
```

Comment:
This part needs to be fixed to detect the latest `pf/*` tag

Reply:
This will need to look at tags, since we don't "release" `pf/vX.Y.Z` according to GH.

Reply:
I might be missing an obvious solution, but I can't seem to find a simple command to list, or filter, tags by name using `gh`.

Reply:
I believe that there is one, but I've been using `git ls-remote` for this:

https://github.com/pulumi/upgrade-provider/blob/cb42bff41add5feada028094f5fc02922fbdbd0b/upgrade/steps-helpers.go#L308-L309